net/transmission/Makefile

```diff
@@ -94,7 +93,7 @@ endef
 define Package/transmission-web
   $(call Package/transmission/template)
   TITLE+= (webinterface)
-  DEPENDS:=transmission-daemon
+  DEPENDS:=transmission-daemon-openssl||PACKAGE_transmission-daemon-polarssl
```

Comment:
When grepping through makefiles I found this example of alternative dependencies:
`./net/pppossh/Makefile: DEPENDS:=+ppp +resolveip @(PACKAGE_dropbear||PACKAGE_openssh-client).`
Would it be better to rewrite it like this?